test/collect.js

```diff
@@ -8,11 +8,8 @@ var AsyncStream = require('../throughs/async')
 //  console.log(err, ary)
 //}))
 
-new ValuesStream([4,5,6]).pipe(new AsyncStream(function (data, cb) {
```

Comment:
is this one failing now or why what is commented out?

Reply:
Oh, this was not even a test. I seems like it was literally just a work-in-progress

Reply:
True :)